code/modules/projectiles/guns/mounted.dm

```diff
@@ -84,6 +84,190 @@
 	)
 	flags_item =  IS_DEPLOYABLE|TWOHANDED|DEPLOYED_NO_PICKUP|DEPLOY_ON_INITIALIZE
 
+//-------------------------------------------------------
+//MG-2005 mounted minigun
+
+/obj/item/weapon/gun/minigunnest
+	name = "\improper MG-2005 mounted minigun"
+	desc = "The MG-2005 mounted minigun is a gun simple in principle it will shoot allot of bullets really fast and will rip through xeno hordes."
```

Comment:
```suggestion
	desc = "The MG-2005 mounted minigun is a gun simple in principle, it will shoot a lot of bullets really fast and will rip through xeno hordes."
```